src/context/AgencyContext.js

```diff
@@ -1,16 +1,67 @@
 import { createContext, useState } from "react";
+import axios from "axios";
+import { APIUrl } from "../config/config";
 
 export const AgencyContext = createContext({});
 
 export default function AgencyContextProvider({ children }) {
-  const [selectedAgency, setSelectedAgency] = useState();
+  const [selectedAgency, setSelectedAgency] = useState({ company_id: 1 });
   const [pop, setPop] = useState(false);
+  const [cityList, setCityList] = useState({});
+  const [error, setError] = useState(null);
+  const [state, setState] = useState({});
+  const header = {
+    headers: {
+      authorization: `Bearer ${localStorage.token}`,
```

Comment:
Maybe import this token using: useAuth() hook. 
Keep things consistent: 
- OR only access localstorage directly (no context for Auth)
- OR have a context for Auth that manages the localstorage

If you use both approaches you are inviting: 🐞🐛🪲

Reply:
👍